ollama/runtime/pom.xml

```diff
@@ -26,7 +26,36 @@
             <artifactId>quarkus-langchain4j-core</artifactId>
             <version>${project.version}</version>
         </dependency>
-
+        <dependency>
+            <groupId>dev.langchain4j</groupId>
```

Comment:
Why this change?

Reply:
I needed it for one of my tests, but it's no longer in use.
I've deleted it (and I'll see if I need some classes from this dependency later 😉).

Reply:
➡️ https://github.com/quarkiverse/quarkus-langchain4j/pull/522/commits/35a077bd3c62352daa97f70c493a2e2d15f0c265

Reply:
@geoand after remove it i remember (or Maven remember to me 😉) why I've needed the add the dependency: it was to import the class `dev.langchain4j.model.ollama.OllamaStreamingChatModel`.


If I well understood, for the supplier I need a builder and this builder is in this class.

Reply:
I confess that at this stage, I don't understand yet why, for some models, some classes are used from langchain4j and other rewritten with duplicated code.

Other thinks I need to understand, for example, is why for some models the langchain4j client is used or a new one is created (like Mistral for example).

I'm trying to understand the architecture of the code, and I think I'm going to make some mistakes, sorry.

Reply:
The general idea was that we try to reuse upstream bit, but for some integrations the upstream code is so mimimal that it just does not bring any benefit to do so.
Such is the case of the Ollama extension

Reply:
yes, I begin to understand this and I agree with you 😉